src/game_inventory.cpp

```diff
@@ -44,6 +51,17 @@ static item_location inv_internal( player &u, const inventory_selector_preset &p
     return inv_s.execute();
 }
 
+item_location game::get_item_from_inventory( player &p, const std::string &title )
+{
+    const std::string msg = &p == &u ? _( "Your inventory is empty." ) :
```

Comment:
`p.is_npc()` would be cleaner. Also, I'm not certain, but some compilers may complain about the ternary 
`? const char *: std::string()`.